tests/test_model.py

```diff
@@ -1389,26 +1415,17 @@ def test_set_presentation_edition(self):
         edition_admin = self._edition(data_source_name=DataSource.LIBRARY_STAFF, with_license_pool=False)
         edition_mw = self._edition(data_source_name=DataSource.METADATA_WRANGLER, with_license_pool=False)
         edition_od, pool = self._edition(data_source_name=DataSource.OVERDRIVE, with_license_pool=True)
- 
+
         edition_mw.primary_identifier = pool.identifier
         edition_admin.primary_identifier = pool.identifier
 
         # set overlapping fields on editions
         edition_od.title = u"OverdriveTitle1"
-        [joe], ignore = Contributor.lookup(self._db, u"Sloppy, Joe")
```

Comment:
I took these contributors out because they weren't changing anything about the test.